lib/brcobranca/boleto/santander.rb

```diff
@@ -50,8 +50,8 @@ def seu_numero=(valor)
       # Dígito verificador do nosso número.
       # @return [String] 1 caracteres numéricos.
       def nosso_numero_dv
-        nosso_numero = numero_documento.to_s.rjust(12, '0') unless numero_documento.nil?
-        nosso_numero.modulo11_2to9
+        nosso_numero = self.numero_documento.to_s.rjust(12,'0') unless self.numero_documento.nil?
```

Comment:
Line is too long. [97/80]<br>Redundant `self` detected.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.